.palantir/revapi.yml

```diff
@@ -874,6 +874,10 @@ acceptedBreaks:
       justification: "Static utility class - should not have public constructor"
   "1.4.0":
     org.apache.iceberg:iceberg-core:
+    - code: "java.class.defaultSerializationChanged"
```

Comment:
This is in the wrong version isn't it?

Reply:
great catch. the initial change and grade revapiAcceptBreak were run a few months ago. will need to re-run it maybe after 1.16 release.

Reply:
@stevenzwu can you please rebase the PR and make sure to run the rev api task that will automatically add this to the correct version in this file?

Reply:
the diff in the file is still messed up

Reply:
fixed. I first copied the `revapi.yml` file from latest main branch. then applied the `revapiAcceptBreak`. There was a couple additional diffs from the revapi action.

```
➜  iceberg git:(issue-9597-manifest-task) ✗ ./gradlew :iceberg-core:revapiAcceptBreak --justification "Serialization across versions is not supported" \
          --code "java.class.defaultSerializationChanged" \
          --old "class org.apache.iceberg.GenericManifestFile" \
          --new "class org.apache.iceberg.GenericManifestFile"
```